src/desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -137,7 +137,7 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
     return unless @validateAcceptConditions()
     analyticsHooks.trigger 'registration:submit-address'
     @loadingLock @$submit, =>
-      (if @validateForm() then Q() else Q.reject('Please review the error(s) above and try again.')).then =>
```

Comment:
I didn't notice this on the ticket, but did we want to get rid of this message too?

Reply:
Yeah I checked with Joanna